sentry/sentry.conf.example.py

```diff
@@ -300,7 +300,6 @@ def get_internal_network():
             "organizations:insights-initial-modules",
```

Comment:
this line exists already.

Reply:
Ah you're right. I didn't notice that. Thanks though!